portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,19 +58,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery queryResults = datastore.prepare(query);
 
-    ArrayList<String> comments = new ArrayList<String>();  
-  
+    ArrayList<CommentObject> comments = new ArrayList<CommentObject>();  
     Iterable<Entity> resultsIteratable = queryResults.asIterable();
     Iterator<Entity> resultsIterator = resultsIteratable.iterator();
     for (int i = 0; i < numberComments; i++) {
       if (resultsIterator.hasNext()) {
-        String commentText = (String) resultsIterator.next().getProperty(TEXT_PARAMETER_KEY);
-        comments.add(commentText);
+        Entity comment = resultsIterator.next();
+        CommentObject newComment = new CommentObject((String) comment.getProperty(TEXT_PARAMETER_KEY), (String) comment.getProperty("image_url"));
```

Comment:
nit: Should most likely add `"image_url"` to a constant to keep things consistent